docker-images/operator/scripts/launch_java.sh

```diff
@@ -12,6 +14,41 @@ function get_gc_opts {
   fi
 }
 
+# Generic formula evaluation based on awk
+calc() {
+  local formula="$1"
+  shift
+  echo "$@" | awk '
+    function round(x) {
+      return int(x + 0.5)
+    }
+    {print '"int(${formula})"'}
+  '
+}
+
+calc_maximum_size_opt() {
+  local max_mem="$1"
+  local fraction="$2"
```

Comment:
Maybe you should call it percentage rather than fraction? As that is what you seem to pass in.